public/css/module.less

```diff
@@ -21,6 +21,15 @@
     }
 }
 
+.comment {
+    background-color: transparent;
+}
+
+img.comment-author {
```

Comment:
Do we have the class `comment-author` for different tags?

Reply:
No, just wanted to maintain the specificity. Will remove it right away.